libr/bin/format/elf/elf.c

```diff
@@ -2135,8 +2135,9 @@ char* Elf_(r_bin_elf_get_arch)(ELFOBJ *bin) {
 	case EM_LANAI:
 		return strdup ("lanai");
 	case EM_VIDEOCORE3:
-	case EM_VIDEOCORE4:
-		return strdup ("vc4");
```

Comment:
Theres no vc3 plugin. Why did you removed the vc4 one? We have no tests but this was working fine

Reply:
Whoops, that's leftover from when I was first figuring out how this worked.  Fixed.